diff --git a/src/slic3r/GUI/GUI.cpp b/src/slic3r/GUI/GUI.cpp
index a54dd43..d290ba8 100644
--- a/src/slic3r/GUI/GUI.cpp
+++ b/src/slic3r/GUI/GUI.cpp
@@ -452,7 +452,7 @@ unsigned int combochecklist_get_flags(wxComboCtrl* comboCtrl)
 {
 	unsigned int flags = 0;
 
-	wxCheckListBoxComboPopup* popup = wxDynamicCast(comboCtrl->GetPopupControl(), wxCheckListBoxComboPopup);
+	wxCheckListBoxComboPopup* popup = dynamic_cast<wxCheckListBoxComboPopup*>(comboCtrl->GetPopupControl());
 	if (popup != nullptr) {
 		for (unsigned int i = 0; i < popup->GetCount(); ++i) {
 			if (popup->IsChecked(i))
@@ -465,7 +465,7 @@ unsigned int combochecklist_get_flags(wxComboCtrl* comboCtrl)
 
 void combochecklist_set_flags(wxComboCtrl* comboCtrl, unsigned int flags)
 {
-	wxCheckListBoxComboPopup* popup = wxDynamicCast(comboCtrl->GetPopupControl(), wxCheckListBoxComboPopup);
+	wxCheckListBoxComboPopup* popup = dynamic_cast<wxCheckListBoxComboPopup*>(comboCtrl->GetPopupControl());
 	if (popup != nullptr) {
 		for (unsigned int i = 0; i < popup->GetCount(); ++i) {
 			popup->Check(i, (flags & (1 << i)) != 0);
