diff --git a/src/slic3r/GUI/AuxiliaryDataViewModel.cpp b/src/slic3r/GUI/AuxiliaryDataViewModel.cpp
index f68f733..a21258e 100644
--- a/src/slic3r/GUI/AuxiliaryDataViewModel.cpp
+++ b/src/slic3r/GUI/AuxiliaryDataViewModel.cpp
@@ -332,8 +332,8 @@ wxDataViewItemArray AuxiliaryModel::ImportFile(AuxiliaryModelNode* sel, wxArrayS
         fs::path src_bfs_path(file_path.ToStdWstring());
         wxString dir_path = m_root_dir;
         if (sel != m_root)
-            dir_path += "\\" + sel->name;
-        dir_path += "\\" + src_bfs_path.filename().generic_wstring();
+            dir_path += wxString("\\") + sel->name;
+        dir_path += wxString("\\") + src_bfs_path.filename().generic_wstring();
 
         boost::system::error_code ec;
         if (!fs::copy_file(src_bfs_path, fs::path(dir_path.ToStdWstring()), fs::copy_option::overwrite_if_exists, ec))
