diff --git a/src/slic3r/GUI/ImageGrid.cpp b/src/slic3r/GUI/ImageGrid.cpp
index 454617e..252b1fc 100644
--- a/src/slic3r/GUI/ImageGrid.cpp
+++ b/src/slic3r/GUI/ImageGrid.cpp
@@ -656,10 +656,12 @@ void Slic3r::GUI::ImageGrid::renderContent1(wxDC &dc, wxPoint const &pt, int ind
         // Draw buttons on hovered item
         wxRect rect{pt.x, pt.y + m_content_rect.GetBottom() - m_buttons_background.GetHeight(), m_content_rect.GetWidth(), m_buttons_background.GetHeight()};
         if (hit) {
-            renderButtons(dc, {_L("Delete"), (wxChar const *) secondAction, thirdAction.IsEmpty() ? nullptr : (wxChar const *) thirdAction, nullptr}, rect,
+            std::vector<wxString> btns{_L("Delete"), secondAction};
+            if (!thirdAction.IsEmpty()) btns.push_back(thirdAction);
+            renderButtons(dc, btns, rect,
                           m_hit_type == HIT_ACTION ? m_hit_item & 3 : -1, states);
         } else if (!nonHoverText.IsEmpty()) {
-            renderButtons(dc, {(wxChar const *) nonHoverText, nullptr}, rect, -1, states);
+            renderButtons(dc, std::vector<wxString>{nonHoverText}, rect, -1, states);
         }
     } else {
         dc.SetTextForeground(*wxWHITE); // time text color
@@ -705,7 +707,7 @@ void Slic3r::GUI::ImageGrid::renderContent2(wxDC &dc, wxPoint const &pt, int ind
     renderIconText(dc, m_model_weight_icon, file.Metadata("Weight", "0g"), rect);
 }
 
-void Slic3r::GUI::ImageGrid::renderButtons(wxDC &dc, wxStringList const &texts, wxRect const &rect2, size_t hit, int states)
+void Slic3r::GUI::ImageGrid::renderButtons(wxDC &dc, std::vector<wxString> const &texts, wxRect const &rect2, size_t hit, int states)
 {
     // Draw background
     {
