diff --git a/src/slic3r/GUI/GCodeRenderer/AdvancedRenderer.cpp b/src/slic3r/GUI/GCodeRenderer/AdvancedRenderer.cpp
index ae7d66e..dcdfd71 100644
--- a/src/slic3r/GUI/GCodeRenderer/AdvancedRenderer.cpp
+++ b/src/slic3r/GUI/GCodeRenderer/AdvancedRenderer.cpp
@@ -330,13 +330,13 @@ namespace Slic3r
                 case EViewType::VolumetricRate:
                 {
                     fprintf(fp, "map_Kd range_color.png\n");
-                    export_image(m_p_color_range_texture, parent_path.wstring() + "/range_color.png");
+                    export_image(m_p_color_range_texture, parent_path.wstring() + L"/range_color.png");
                     break;
                 }
                 case EViewType::FeatureType:
                 {
                     fprintf(fp, "map_Kd Extrusion_Role_Colors.png\n");
-                    export_image(m_p_role_colors_texture, parent_path.wstring() + "/Extrusion_Role_Colors.png");
+                    export_image(m_p_role_colors_texture, parent_path.wstring() + L"/Extrusion_Role_Colors.png");
                     break;
                 }
                 case EViewType::Tool:
@@ -344,7 +344,7 @@ namespace Slic3r
                 case EViewType::ColorPrint:
                 {
                     fprintf(fp, "map_Kd tool_colors.png\n");
-                    export_image(m_p_tool_colors_texture, parent_path.wstring() + "/tool_colors.png");
+                    export_image(m_p_tool_colors_texture, parent_path.wstring() + L"/tool_colors.png");
                     break;
                 }
                 }
